### DISCUSSION TITLE
GitHub Beta

### DISCUSSION THREAD

**Discussion by iamrithiks** [github/india#19] [Satellite India 2021 Q&A] [answered]
Do we have other GitHub features which were not introduced in this event but are available in beta?

**Comment by AnandChowdhary** [comment, marked as answer]
You can find features available to you in "Feature preview": https://docs.github.com/en/github/getting-started-with-github/exploring-early-access-releases-with-feature-preview. I think this is what you mean with "features available in beta".

Related: [GitHub Stars](https://stars.github.com) also gives you access to early feature before their public betas. If you know anyone doing great work in open-source, please nominate them!
